static/js/biomedical/protein/protein_protein_interaction_graph.tsx

```diff
@@ -103,7 +108,26 @@ export class ProteinProteinInteractionGraph extends React.Component<
     }
     return (
       <>
-        <div id={CHART_ID}></div>
+        <div id={GRAPH_ID}></div>
+        <ProteinProteinInteractionTable
+          columns={[
```

Comment:
this looks like a constant and so probably doesn't need to be passed as a prop?

Reply:
got it will move to table component!